lib/health_monitor/providers/elastic_search.rb

```diff
@@ -15,17 +15,17 @@ def initialize
           @ping_url = nil
         end
 
-        def ping_url
-          @ping_url ||= self.elastic_search_url + '/_cluster/health'
+        def elastic_ping_url
+          @ping_url ||= @elastic_search_url + '/_cluster/health'
```

Comment:
Instead of self. or @ can you just remove the @  I want to use the accessor here not the variable directly